tf-static-website/23-cloudfront-frontend.tf

```diff
@@ -0,0 +1,65 @@
+resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
+  comment  = "This is used for my website."
+  provider = aws.ops-blog
+}
+
+locals {
+  s3_origin_id = "myS3Origin"
+}
+
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  origin {
+    domain_name = aws_s3_bucket.ops_website.bucket_regional_domain_name
+    origin_id   = local.s3_origin_id
+
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.origin_access_identity.cloudfront_access_identity_path
+    }
+  }
+
+  enabled             = true
+  comment             = "This is used for my website."
+  default_root_object = "index.html"
+
+  logging_config {
+    include_cookies = false
+    bucket          = aws_s3_bucket.ops_website.bucket_regional_domain_name
+    prefix          = "access_log"
+  }
+
+  aliases = [ var.domain_name ]
+
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
```

Comment:
**POST** method may be used later. It would be nice to add here in advance, even though you don't have any plan for that. Because it takes quite a time to deploy cloudfront.

Reply:
I'll add POST as well!